tests/driver_hsc/Makefile

```diff
@@ -0,0 +1,7 @@
+include ../Makefile.tests_common
+
+USEMODULE += hsc
+USEMODULE += xtimer
+FEATURES_REQUIRED += periph_i2c
```

Comment:
```suggestion
```

The driver itself should require `periph_i2c`, so no need to ask for it again here.